contracts/stake/src/contract.rs

```diff
@@ -360,6 +395,33 @@ impl StakingTrait for Staking {
         WithdrawableRewardsResponse { rewards }
     }
 
+    fn query_withdrawable_rewards_ch(
+        env: Env,
+        user: Address,
+        chunk_size: u32,
+        start_day: Option<u64>,
+    ) -> WithdrawableRewardsResponse {
+        env.storage()
+            .instance()
+            .extend_ttl(INSTANCE_LIFETIME_THRESHOLD, INSTANCE_BUMP_AMOUNT);
+
+        let stakes = get_stakes(&env, &user);
+        let mut rewards = vec![&env];
+
+        // Iterate over all distributions and calculate withdrawable rewards
+        for asset in get_distributions(&env) {
+            let (pending_reward, _) =
+                calculate_pending_rewards_chunked(&env, &asset, &stakes, chunk_size, start_day);
```

Comment:
why don't we return just `pending_reward` since we're dropping the 2nd part of the tuple with `_` from memory

Reply:
It's used in another call of that function.